dart/dynamics/ReferentialSkeleton.hpp

```diff
@@ -54,6 +55,15 @@ class ReferentialSkeleton : public MetaSkeleton
   /// Default destructor
   virtual ~ReferentialSkeleton() = default;
 
+  // Documentation inherited
+  std::unique_ptr<common::MutexReference> getCustomMutex() const override;
+
+  /// Returns sorted mutexes in order of memory addresses.
+  ///
+  /// \note Generally, you don't need to use this function in favor of
+  /// getCustomMutex() that guarantees deadlock-free locking.
+  const std::set<std::mutex*>& getStdMutexes() const;
```

Comment:
Do we really want to expose this function at all? It strikes me as needlessly exposing implementation detail to the user. E.g. what if we decide later to not use `std::set` to store the mutexes? Then this function that returns a const-reference would have to be removed, or else we'd need to hang onto a separate `std::set` copy of the mutexes. It's hard for me to imagine a situation where a user would want access to the individual mutexes instead of the `MutexReference`.

Reply:
I agree with @mxgrey. I suggest we remove this unless we have a specific use-case in mind that requires it.

Reply:
Removed.